app/providers/atmosphere/optimization_strategy/default.rb

```diff
@@ -33,12 +33,13 @@ def can_scale_manually?
       protected
 
       def vmt_candidates_for(appliance)
-        VirtualMachineTemplate.where(
+        vmts = VirtualMachineTemplate.where(
           appliance_type: appliance.appliance_type,
           state: 'active',
-          tenant_id:
-            appliance.tenants.active.funded_by(appliance.fund)
         )
+        vmts.select do |vmt|
```

Comment:
If validation checking if VMT is assigned to at least one Tenant will be present this code most probably will not be necessary